R/as_rtf.R

```diff
@@ -112,3 +124,122 @@ as_rtf.gsBinomialExactTable <- function(
 
   invisible(x_out)
 }
+#' @rdname as_rtf
+#' @param title Title of the report.
+#' @param footnote_p_onesided Footnote for one-side p-value. 
+#' @param footnote_appx_effect_at_bound Footnote for approximate effect treatment at bound.
+#' @param footnote_p_cross_hr1 Footnote for cumulative type I error.
+#' @param footnote_p_cross_hr0.5 Footnote for cumulative power under the alternate hypothesis.
+#' @param footnote_specify Vector of string to put footnote super script.
+#' @param footnote_text Vector of string of footnote text.
+#' @importFrom r2rtf rtf_title rtf_footnote
+#' @export
+as_rtf.gsBoundSummary <- function(
+    x,
+    file,
+    ...,
+    title = "Boundary Characteristics for Group Sequential Design",
+    footnote_p_onesided = "one-side p-value for experimental better than control",
+    footnote_appx_effect_at_bound = NULL,
+    footnote_p_cross_hr1 = "Cumulative type I error assuming binding futility bound",
+    footnote_p_cross_hr0.5 = "Cumulative power under the alternate hypothesis",
```

Comment:
The argument `footnote_p_cross_hr0.5` is not proper. It happens that the HR can be some values other than 0.5. Please name it as `footnote_p_cross_h0` and `footnote_p_cross_h1`.